ci/scripts/move_to_production.py

```
@@ -0,0 +1,224 @@
+    """
+
+    variables = {
+        "org": args.org,
+        "number": args.project
+    }
+
+    result = graphql_request(query, variables)
+
+    if not result.get("data", {}).get("organization", {}).get("projectV2"):
+        print(f"Project {args.project} not found in organization {args.org}")
+        exit(1)
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)